### PR TITLE
Remove code that would cause accessibility header role to be spoken twice

### PR DESCRIPTION
**Pull request by KevinGVargas**
## Summary
The header role is being said twice in android. Stopped that from happening.

## Changelog
[General] [Added] - removed code that would cause accessibility header role to be spoken twice

## Test Plan
Test plan is testing in RNTester making sure the examples work

## Note:
generatedComponentApiDocs.js was modified by lint commands. I can submit a PR without it if preferred.

### CODE REVIEW

**Comment by MahendraBalu** [comment]
Please let me know which version of react-native version this fix will be available. I am currently at "react-native": "^0.61.5" version.

**Comment by KevinGVargas** [comment]
@MahendraBalu sorry im uncertain on what release this was put in, if possible, using the latest version will probably contain it.

File: ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java [hunk 1]
```diff
@@ -367,9 +367,8 @@ public static void setRole(
     } else if (role.equals(AccessibilityRole.SUMMARY)) {
       nodeInfo.setRoleDescription(context.getString(R.string.summary_description));
     } else if (role.equals(AccessibilityRole.HEADER)) {
-      nodeInfo.setRoleDescription(context.getString(R.string.header_description));
```
Comment (elfstyle) [inline ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java:370]: This change breaks accessibilityRole='header' on Android for RN 63.x. It was working fine on 62.2

Comment (nico22utn) [reply ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java:370]: Yep! That's right! I'm having this issue on my RN app.

**Pull request closed without merging**